src/components/ItemSelector/SelectedItems.js

```diff
@@ -242,7 +244,10 @@ export class SelectedItems extends Component {
                     </Droppable>
                 </DragDropContext>
                 <div className="deselect-all-button">
-                    <Button onClick={this.onDeselectAll}>
+                    <Button
+                        onClick={this.onDeselectAll}
+                        dataTest={`${this.props.dataTest}-deselect-all-button`}
```

Comment:
Based on the decision (https://github.com/dhis2/notes/blob/master/decisions/2020/11/10-cypress-meeting.md#naming), I'm wondering if the format should instead be:
`-button-deselectall`
So, the component first, then the modifier follows it. Also, suggested but no in the "decisions" try to reserve hyphens for component separators.